src/common/meta/src/cache_invalidator.rs

```diff
@@ -14,19 +14,23 @@
 
 use std::sync::Arc;
 
+use table::metadata::TableId;
+
 use crate::error::Result;
-use crate::ident::TableIdent;
+use crate::table_name::TableName;
 
 /// Places context of invalidating cache. e.g., span id, trace id etc.
-#[derive(Debug, Default)]
+#[derive(Default)]
 pub struct Context {
     pub subject: Option<String>,
 }
 
 #[async_trait::async_trait]
 pub trait CacheInvalidator: Send + Sync {
     // Invalidates table cache
-    async fn invalidate_table(&self, ctx: &Context, table_ident: TableIdent) -> Result<()>;
+    async fn invalidate_table_id(&self, ctx: &Context, table_id: TableId) -> Result<()>;
+
+    async fn invalidate_table_name(&self, ctx: &Context, table_name: TableName) -> Result<()>;
```

Comment:
Unfortunately, The `TableInfo` also contains the name attribute.

Reply:
I think the table name in `TableInfo` should be removed. It should simply contain table id. I will leave a todo for myself.